src/common/get-extension-id.ts

```diff
@@ -1,7 +1,11 @@
+import browser from 'webextension-polyfill';
+
 export default function getExtensionId(): string | null {
-  const chrome: any = (global as any).chrome;
-  if (chrome && chrome.extension && chrome.extension.getURL) {
-    return chrome.extension.getURL('');
+  try {
+    return browser.runtime.id;
+  } catch (error) {
+    console.error('Failed to get extension id');
   }
```

Comment:
I'm a little trepidatious about `webextension-polyfill` would break our Safari support. Specifically, [this issue](https://github.com/mozilla/webextension-polyfill/issues/364).

This draft block will trigger a `console.error` for Safari extensions using the SDK. 

```suggestion
  return chrome?.extension?.getURL?.('') ?? browser?.runtime?.id;
```

Reply:
Since I've removed webextension-polyfill the code now uses `browser.runtime.id` (chrome.runtime.id).
Also `chrome.extension.getURL` is deprecated in MV3 [(See here)](https://developer.chrome.com/docs/extensions/reference/api/extension#method-getURL).